tests/pck/023.0102.pck

```diff
@@ -4,6 +4,5 @@ RSI5B:12345678901A:1604
 0001 00000000002
 0011 00000010416
 0012 00000311016
-0020 00001800000
```

Comment:
Test data shouldn't change, we need these to be made. The new schemas must be slightly wrong.